### PR TITLE
20721 - Client changes from EFT to PAD

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:*
https://github.com/bcgov/entity/issues/20721
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/common/PaymentMethods.vue [hunk 1]
```diff
@@ -17,7 +17,6 @@
         class="payment-card py-8 px-8 mb-4 elevation-1"
         :class="{'selected': isPaymentSelected(payment)}"
         :data-test="`div-payment-${payment.type}`"
-        @click="paymentMethodSelected(payment)"
```
Comment (Jxio) [inline auth-web/src/components/auth/common/PaymentMethods.vue:20]: redundant, it always send two requests

File: auth-web/src/util/constants.ts [hunk 2]
```diff
@@ -100,7 +100,7 @@ export enum Pages {
     STAFF_DASHBOARD_INVITATIONS = '/staff/dashboard/invitations',
     STAFF_DASHBOARD_SUSPENDED = '/staff/dashboard/suspended',
     STAFF_DASHBOARD_INACTIVE = '/staff/dashboard/inactive',
-    MAKE_PAD_PAYMENT = '/make-cc-payment/',
+    MAKE_CC_PAYMENT = '/make-cc-payment/',
```
Comment (Jxio) [inline auth-web/src/util/constants.ts:103]: it's pay for outstanding balance by credit card, but says `MAKE_PAD_PAYMENT`

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2941-fp9gks8g.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2941-fp9gks8g.web.app

**Comment by seeker25** [comment]
Why is paying for EFT via PAD an option? 

are you creating the CFS account for PAD.. waiting for the confirmation period to be over.. re-creating the invoices in PAD?
and reversing the invoices for EFT?

**Review by seeker25 — CHANGES_REQUESTED** [review]
Pay EFT balance with PAD, I don't think this works

**Comment by ochiu** [comment]
Agreed, I believe we decided we will just go with the CC flow as we removed Option #2 from the ticket to pay with PAD.

**Comment by Jxio** [comment]
I added feature flag of pay EFT balance by PAD.
Currently when user change from EFT to PAD, if it has remaining balances, it direct to pay outstanding balance screen, it only has cc option cause we don't have 'enable-eft-balance-by-pad' for now.

**Comment by Jxio** [comment]
closing this ticket, will open another one for removing pad option

**Pull request closed without merging**